cypress/support/e2e.js

```diff
@@ -0,0 +1 @@
+import './commands'
```

Comment:
```suggestion
import './commands'

```

Recomenda-se deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como o GitHub não exibem o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção a ser seguida.